src/util/ast.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
```

Comment:
Consider `import * as jsonc from 'jsonc-parser'` so that the similar-sounding things are more clearly distinguished, and we don't need to pick aliases.